_data/health.yml

```diff
@@ -87,7 +87,7 @@ websites:
     tfa:
       - sms
       - phone
-      - software
+      - totp
```

Comment:
```suggestion
      - totp
      - proprietary
```

Reply:
Which proprietary software (that doesn't just support regular totp, like Authy) do you mean?

Reply:
@phallobst You can use the "Google App" on your phone to sign in to your account, which is `proprietary` - because you don't enter a TOTP but just answer the push message on the phone to log in. I think that's a fair use of prop ;)

Reply:
I should have known that, since I also use push notifications for Google 2FA. Oh well...